docs/io.cozy.calendar.md

```diff
@@ -0,0 +1,54 @@
+[Table of contents](README.md#table-of-contents)
+    
+# Cozy Calendar doctype
+
+## `io.cozy.calendar`
+
+The `io.cozy.calendar` doctype is loosely based on the [iCalendar RFC](https://datatracker.ietf.org/doc/html/rfc5545), in that some of the attributes have been renamed for clarity. The attributes with a `?` are optional.
```

Comment:
I would have expected an `io.cozy.calendar` to be a thing that regroups events, like "School schedule" or "Personal"), and what is described here to be an `io.cozy.calendar.event`.

Reply:
Yes, the `io.cozy.calendar` is a "parent" doctype, including io.cozy.calendar.events and io.cozy.calendar.todos

Reply:
Adressed in this commit -> https://github.com/cozy/cozy-doctypes/commit/5d4bc8ab3ee501ad97bd9cb2c3ee25a77954e325